verilog/tools/ls/autoexpand.cc

```diff
@@ -648,20 +656,27 @@ void AutoExpander::Module::EmitUnconnectedOutputRegDeclarations(
 }
 
 void AutoExpander::Module::GenerateConnections(
-    const Template *tmpl,
+    absl::string_view instance_name, const Template *tmpl,
     const std::function<void(const Port &, const Connection &)> &fun) const {
   for (const Port &port : ports_) {
     Connection connected{.port_name = port.name, .emit_dimensions = true};
     if (tmpl) {
       const auto it = tmpl->connections.find(port.name);
       if (it != tmpl->connections.end()) connected = it->second;
     }
+    size_t pos = connected.port_name.find('@');
+    while (pos != std::string::npos) {
+      connected.port_name.replace(pos, 1, instance_name.begin(),
+                                  instance_name.length());
+      pos = connected.port_name.find('@', pos);
+    }
+    connected.port_name = connected.port_name;
```

Comment:
this looks ... unnecessary

Reply:
It does :rofl: not sure what happened there.